.github/pull_request_template.md

```diff
@@ -1,30 +1,29 @@
 ## Warning!!
 
 The AP CSP Create Performance Task is in progress from April 1 - May 1 2023. Please
-consider any risk introduced by this PR that could affect our students taking AP CSP. Code.org
-students taking AP CSP primarily use App Lab for their Create Task, however a small percent use Game
-Lab. Carefully consider whether your change has any risk of alterering, changing, or breaking
-anything in these two labs. Even small changes, such as a different button color, are considered
-significant during this time period. Reach out to the Student Learning team or Curriculum team for
-more details.
+consider any risk introduced by this PR that could affect our students taking AP CSP.
+Code.org students taking AP CSP primarily use App Lab for their Create Task, however a
+small percent use Game Lab. Carefully consider whether your change has any risk of
+alterering, changing, or breaking anything in these two labs. Even small changes, such as
+a different button color, are considered significant during this time period. Reach out to
+the Student Learning team or Curriculum team for more details.
 
 <!-- end warning -->
 
 <!--
   A summary of the change, including any relevant background, motivation, and context.
-  If relevant, include a description, screenshots, and/or video of the existing and new behavior.
+  If relevant, include a description, screenshots, and/or video of the existing and new
+  behavior.
 -->
 
 ## Links
```

Comment:
Any content in a Pull Request could contain a link to some other resource. It seems strange to have a dedicated “Links” section. I *do* try to avoid writing up a Pull Request Summary that duplicates information already in JIRA, so maybe we should include the suggestion to reference the JIRA and/or Slack discussion in the Summary section?

Reply:
The one thing I changed about the Links section was to uncomment the Jira ticket reference to strongly encourage linking to it. Personally I'd be a big fan of Jira Issue numbers in Pull Request names and/or commit messages.

Interesting comment on avoiding duplicating Jira. I see the PR as the best source of truth for the "Why" behind changes - I'd rather have a link to a PR from an empty Jira ticket than have a link to Jira from an empty PR.